Tests/AdyenTests/Adyen Tests/UI/DropIn/DropInActionTests.swift

```diff
@@ -20,11 +20,10 @@ class DropInActionsTests: XCTestCase {
 
     func testOpenRedirectAcitionOnDropIn() {
         let config = DropInComponent.PaymentMethodsConfiguration(clientKey: Dummy.dummyClientKey)
-        config.environment = .test
+        config.payment = Payment(amount: Payment.Amount(value: 100, currencyCode: "CNY"), countryCode: "CN")
 
         let paymenMethods = try! JSONDecoder().decode(PaymentMethods.self, from: DropInTests.paymentMethods.data(using: .utf8)!)
         sut = DropInComponent(paymentMethods: paymenMethods, paymentMethodsConfiguration: config)
-        sut.payment = Payment(amount: Payment.Amount(value: 100, currencyCode: "CNY"), countryCode: "CN")
```

Comment:
you should set the environment on `sut` to `.test`

Reply:
👍